internal/osutils/shortcut/shortcut_windows.go

```diff
@@ -0,0 +1 @@
+package shortcut
```

Comment:
It appears that the `shortcut` package is only being used from Windows-specific files. So either this file is no longer necessary or the contents of `shortcut.go` should be here.

Reply:
That's just because the current iteration is only Windows focused. The distinction is necessary because the windows specific code won't compile outside of windows. And if the package has no Go files on other platforms then building it will fail outright.